1-exercises/B-object-arrays/exercise-2.js

```diff
@@ -16,7 +16,14 @@ let store = [
 
 let money = 1.5;
 
-let thingsICanBuy =
+let thingsICanBuy =  []
```

Comment:
I think you don't need to declare this as an empty array before

Reply:
agree, removed it